database/Seeders/eggs/voice-servers/egg-teamspeak3-server.json

```diff
@@ -53,7 +55,38 @@
             "default_value": "10011",
             "user_viewable": true,
             "user_editable": false,
-            "rules": "required|integer|between:1,65535"
+            "rules": "required|integer|between:1,65535",
```

Comment:
Ports below 1024 can't be used as they require root. This applies to all of the variables below. TS3 also doesn't try to bind to ports below 1024.

Reply:
Changed